jdaviz/configs/default/plugins/metadata_viewer/metadata_viewer.vue

```diff
@@ -2,7 +2,7 @@
   <j-tray-plugin
     :description="docs_description || 'View metadata.'"
     :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#metadata-viewer'"
-    :popout_button="popout_button"
+    :popout_button="popout_button"

```

Comment:
did anything change here?

Reply:
It might be a line break character getting automatically removed/standardized by git.